### PR TITLE
Rdap refactoring

### PR DESCRIPTION
**Pull request by eshryane**

### CODE REVIEW

File: whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java [hunk 1]
```diff
@@ -417,19 +405,20 @@ private IpEntry lookupParentIpEntry(final IpInterval ipInterval) {
...
+                throw new IllegalStateException("No parent for " + ipInterval);
             }
             return firstLessSpecific.get(0);
-        }
-
-        if (ipInterval instanceof Ipv6Resource) {
-            final List<Ipv6Entry> firstLessSpecific = ipv6Tree.findFirstLessSpecific((Ipv6Resource) ipInterval);
-            if (firstLessSpecific.isEmpty()) {
-                throw new IllegalStateException("No parent for " + ipInterval.toString());
+        } else {
```
Comment (MiguelAHM) [inline whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java:411]: here maybe it is more clear "else if()" than "else { if ()" ??

File: whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapObjectMapperTest.java [hunk 2]
```diff
@@ -1,798 +0,0 @@
-package net.ripe.db.whois.api.rdap;
```
Comment (MiguelAHM) [inline whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapObjectMapperTest.java:1]: why have you removed this test?

**Review by MiguelAHM — APPROVED** [review]

File: whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapObjectMapperTest.java (hunk 2 above)
Comment (eshryane) [reply whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapObjectMapperTest.java:1]: Yes, its completely obsolete, I confirmed all the tests were moved to integration tests.

File: whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java (hunk 1 above)
Comment (eshryane) [reply whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java:411]: I didn't want a dangling else at the bottom, so added all braces to be really clear.

**Pull request merged**